src/Teams/Services/ManageTeamsMembersService.cs

```diff
@@ -0,0 +1,13 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Teams.Services
+{
+    public class ManageTeamsMembersService : IManageTeamsMembersService
```

Comment:
It seems that the requested service will contain business logic and processes and the better place for them will be a separate .netstandard 2.1 class library project. Please cooperate with @desenzo as he faced with the same issue and to prevent conflicts during merge. Using separate library have a lot of benefits as for example you can reuse this library in the next projects if you want to share same business ideas between them.